src/edu/csus/ecs/pc2/ui/EditProblemPane.java

```diff
@@ -1129,6 +1129,8 @@ private Problem getProblemFromFields(Problem checkProblem, ProblemDataFiles data
         checkProblem.setActive(!getDeleteProblemCheckBox().isSelected());
         checkProblem.setShortName(getShortNameTextfield().getText());
         
+        checkProblem.setOnLoadDataFilesLoadSamplesFirst(multipleDataSetPane.isLoadSamplesFirst());
```

Comment:
Note that in MultipleDataSetPane, the method(s) (analogous to the comments about class `Problem`, above) ARE named simply and more clearly -- `isLoadSamplesFirst()` is much clearer than the (current) method names in `Problem`.

(No action required on this comment; just pointing it out...)

Reply:
pushed fix.